### DISCUSSION TITLE
camera wobble back and forth (way worst when overlay are added)

### DISCUSSION THREAD

**Discussion by pailheres** [SauceLLC/sauce4zwift#69] [General]
**Describe the bug**
Im always on Camera 1 or Camera 2 in zwift.
I have been using Sauce for 3 weeks only.
I have notice since last zwift update that the camera wobble back and forth. I cannot tell if it was there before but now it is worst or if it is a new problem. The back and forth frequency is something around 1 or 2 second
It is way worst when in a big pack.
It is also worst when adding the overlays. With only the sauce taskbar, it is barely noticeable.
My laptop and my desktop are doing the same thing and i have tried it on three different monitor.
I tried to change resolution in zwift setting and the problem is there on all settings.

I liked your sauce overlay a lot during races.
But since last week i have to hide overlays and only use the sauce taskbar for my info, because the back and forth of the camera is too much for me.

**To Reproduce**

**Expected behavior**

**Screenshots**
I will try to do a screen video of this and share it somewhere

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome version 109.0.5414.120
 - sauce Version 0.12.0

**Additional context**

**Comment by mayfield** [comment]
Hi @pailheres 

This is ol' rubber banding issue with Zwift. Basically any extra load on the system can induce this sea sickness if the CPU or GPU are strapped.

Couple things to try:

1. There are some tips in this post about how to make Zwift higher priority that might help, https://zwifthacks.com/a-simple-way-to-fight-fps-dropping-and-rubber-banding/
2. If you don't have "GPU Enabled" in the settings (it defaults off because of some system compatibility), try and enable that and see if it helps (and is compatible with your system).

If those don't help you might try reducing the number of Sauce windows or go into their settings and increase the delay between updates.  The groups window and gauges have a refresh delay setting for example that could reduce load and thus reduce rubber banding.
